scripts/msct_moco.py

```diff
@@ -18,58 +18,55 @@
 # TODO: do not output inverse warp for ants
 # TODO: ants: explore optin  --float  for faster computation
 
+import glob
 import os
+import shutil
 import sys
-import commands
+
 import numpy as np
+
+import sct_apply_transfo
+import sct_maths
 import sct_utils as sct
 from msct_image import Image
-from sct_image import split_data
```

Comment:
I have start removing "from ... import ..." paradigm more systematically, but I guess there is no emergency